pkg/util/ipset/ipset.go

```diff
@@ -243,8 +243,8 @@ func (runner *runner) FlushSet(set string) error {
 
 // DestroySet is used to destroy a named set.
 func (runner *runner) DestroySet(set string) error {
-	if _, err := runner.exec.Command(IPSetCmd, "destroy", set).CombinedOutput(); err != nil {
-		return fmt.Errorf("error destroying set %s:, error: %v", set, err)
+	if out, err := runner.exec.Command(IPSetCmd, "destroy", set).CombinedOutput(); err != nil {
+		return fmt.Errorf("error destroying set %s, error: %v (%s)", set, err, out)
```

Comment:
If we revisit it, perhaps we should return error directly instead of wrapping error.

And, please tell us why you return `out`?

Reply:
I have test it, command `runner.exec.Command(IPSetCmd, "destroy", set).CombinedOutput()`  return `[]byte` and `error`, and `[]byte` is `The set with the given name does not exist`, while `error` is `exit status 1`. We just use `error` missing important messages.  Besides, in my local, i destroy a non-exist ipset, the return messages is `The set with the given name does not exist`. So i think we should use `[]byte` also. WDYT?

Reply:
Oh, thanks for catching it.